packages/execute-driver/test/e2e/plugin-e2e-specs.js

```diff
@@ -0,0 +1,154 @@
+import path from 'path';
+const APPIUM_HOME = path.resolve(__dirname, '..', '..', '..', 'local_appium_home');
+// need to assign this before appium is imported via e2eSetup
+process.env.APPIUM_HOME = APPIUM_HOME;
+
+import { e2eSetup } from '@appium/base-plugin/build/test/helpers';
+import { remote as wdio } from 'webdriverio';
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { W3C_ELEMENT_KEY, MJSONWP_ELEMENT_KEY } from '../../lib/execute-child';
+
+chai.use(chaiAsPromised);
+const should = chai.should();
+
+const THIS_PLUGIN_DIR = path.resolve(__dirname, '..', '..', '..');
+const TEST_HOST = 'localhost';
+const TEST_PORT = 4723;
```

Comment:
use [get-port](https://npm.im/get-port)

Reply:
this turned out to be problematic. get-port is async and it doesn't fit well because i am already overwriting a mocha before using my helper function? but @boneskull you being a mocha person probably know how to do this right.

Reply:
i'd probably just call `get-port` from `e2eSetup()`.   allow a `port` param still, but if it's `undefined`, use `getPort()`.

Reply:
you could also use a getter, I think.

```
let gotPort;
before(async function() {
  gotPort = await getPort()
});

describe('stuff', function() {
  e2eTest({...opts, get port() { return gotPort }})
});

Reply:
whenever i try to import 'get-port' I get

```
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/jlipps/Code/appium/appium-plugins/packages/execute-driver/node_modules/get-port/index.js from /Users/jlipps/Code/appium/appium-plugins/packages/execute-driver/build/test/e2e/plugin-e2e-specs.js not supported.
Instead change the require of index.js in /Users/jlipps/Code/appium/appium-plugins/packages/execute-driver/build/test/e2e/plugin-e2e-specs.js to a dynamic import() which is available in all CommonJS modules.
```

This is true even if I make it a dynamic `await import` as it recommends. Feels like all sindresorhus's stuff is incompatible with our code in this way now that he's made it all ESM only? But maybe you know a way to make it compatible?